### ISSUE TITLE
Property 'changes' is missing when following the docs

### ISSUE THREAD

**Issue by TheRedD0t** [ngrx/platform#399]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[X] Documentation issue or request
</code></pre>

## What is the current behavior?
When following the docs for the [Entity Adapter/Adapter Collection Methods](https://github.com/ngrx/platform/blob/c31573f1c4d9cc02a90cd2033f68d83b17b74969/docs/entity/adapter.md#adapter-collection-methods), I get a `"Property 'changes' is missing in type 'User'."` error at:
```
[...]
case UserActions.UPDATE_USER: {
  return adapter.updateOne(action.payload.user, state);
}

case UserActions.UPDATE_USERS: {
  return adapter.updateMany(action.payload.users, state);
}
[...]
```
But the docs give me no information that I should add this property to the model.
## Expected behavior:
The docs should describe, that I have to add it to the model.

## Minimal reproduction of the problem with instructions:
Follow the docs for the [Entity Adapter/Adapter Collection Methods](https://github.com/ngrx/platform/blob/c31573f1c4d9cc02a90cd2033f68d83b17b74969/docs/entity/adapter.md#adapter-collection-methods).
_________
Sorry for the sparse issue description, I don't have much time at the moment.

**Comment by brandonroberts** [comment]
Yes, this is an error in the docs. The `updateOne` method takes an object of `{ id: string, changes: T }` and the `updateMany` takes an array of the same object `{ id: string, changes: T }[]`.
